webapp/handlers.py

```diff
@@ -114,6 +120,13 @@ def clear_trailing():
 
     @app.before_request
     def prometheus_metrics():
+        # Accept-encoding counter
+        accept_encoding_counter.labels(
+            accept_encoding=flask.request.headers.get("Accept-Encoding"),
+            user_agent=flask.request.headers.get("User-Agent"),
```

Comment:
so actually the user agent will be an issue. 

The way we can gather prometheus metrics is with a  determined number of labels, otherwise it becomes a nightmare to aggregate values and we will fill the database very fast.

By doing a quick check on graylog currently snapcraft has <316,287 different user agent values just in the last 7 days.

instead of storing the user agent like this, could we determine/calculate what browser is being used from the user agent and store the browser name instead as a label?
As an example 
```
accept_encoding_counter{accept_encoding="br",browser="Chrome"} 12
accept_encoding_counter{accept_encoding="br",browser="Firefox"} 9
```

I will make it much easier for us to do some analytics.

Reply:
Good point, thanks!

I've changed it to only send "browser_family" and "browser_version". Hopefully this is a small enough variation to handle. Or do you need me not to include version? I just thought there's a good chance that the user agent string changed between versions of browser, and it would be good to detect that so we can inform the world.